### DISCUSSION TITLE
Disable autoload, start, etc

### DISCUSSION THREAD

**Discussion by ghost** [trunk-io/docs#312] [Q&A]
I can't seem to find where these live in the docs. How do I disable trunk from starting the daemon via the yaml config?

**Comment by det** [comment]
Hi @chrisgminted,

We don't currently have a way to prevent the daemon from starting when using trunk as all operations are run through the daemon. If you would like to manually shut it down, you can do so with `trunk daemon shutdown`. It will also shut down on its own after 3 hours of inactivity. Please let us know if you solves your problem.
